Jenkinsfile

```diff
@@ -30,18 +29,72 @@ pipeline {
 		stage('Setup: Back up DB'){
 			steps{
 				script{
-          withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
-            def release_current_before_biomodels_dump = "${env.RELEASE_CURRENT}_${currentRelease}_before_biomodels.dump"
-            sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_before_biomodels_dump"
-            sh "gzip -f $release_current_before_biomodels_dump"
-          }
+					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]){
+						def release_current_before_biomodels_dump = "${env.RELEASE_CURRENT}_${currentRelease}_before_biomodels.dump"
+						sh "mysqldump -u$user -p$pass ${env.RELEASE_CURRENT} > $release_current_before_biomodels_dump"
+						sh "gzip -f $release_current_before_biomodels_dump"
+          				}
+				}
+			}
+		}
+		stage('Setup: Create reactome database from release_current') {
+			steps{
+				script{
+					withCredentials([usernamePassword(credentialsId: 'mySQLUsernamePassword', passwordVariable: 'pass', usernameVariable: 'user')]) {
+						sh "mysql -u$user -p$pass -e \'drop database if exists ${env.REACTOME}; create database ${env.REACTOME}\'"
```

Comment:
Similar to analysis-core PR (https://github.com/reactome/release-jenkins-utils/pull/3#discussion_r436200985) - maybe `${env.REACTOME_DB}` in place of `${env.REACTOME}`